src/solvers/dgsem_tree/indicators_2d.jl

```diff
@@ -185,6 +185,100 @@ function (löhner::IndicatorLöhner)(u::AbstractArray{<:Any,4},
 end
 
 
+# this method is used when the indicator is constructed as for shock-capturing volume integrals
+function create_cache(indicator::Type{IndicatorIDP}, equations::AbstractEquations{2}, basis::LobattoLegendreBasis, number_bounds)
+  ContainerShockCapturingIndicator = Trixi.ContainerShockCapturingIndicatorIDP{real(basis)}(0, nnodes(basis), number_bounds)
+
+  cache = (; ContainerShockCapturingIndicator)
+
+  return cache
+end
+
+function (indicator_IDP::IndicatorIDP)(u::AbstractArray{<:Any,4}, semi, dg::DGSEM, t, dt; kwargs...)
+  @unpack alpha = indicator_IDP.cache.ContainerShockCapturingIndicator
+  alpha .= 0.0
```

Comment:
```suggestion
  alpha .= zero(eltype(alpha))
```
For consistency with the code below?